package.json

```diff
@@ -1,63 +1,66 @@
-        "grunt-postcss": "0.3.0",
-        "grunt-contrib-requirejs": "0.4.1",
-        "grunt-contrib-uglify": "0.2.0",
-        "grunt-contrib-watch": "0.4.3",
-        "grunt-git": "^0.3.4",
-        "grunt-jasmine-node": "0.1.0",
-        "grunt-lib-phantomjs": "0.3.0",
-        "grunt-npm": "git://github.com/sedge/grunt-npm.git#branchcheck",
-        "grunt-shell": "^1.1.2",
-        "grunt-targethtml": "0.2.6",
-        "grunt-template-jasmine-requirejs": "0.1.0",
-        "grunt-text-replace": "^0.4.0",
-        "grunt-update-submodules": "^0.4.1",
-        "grunt-usemin": "0.1.11",
-        "habitat": "^3.1.2",
-        "jasmine-node": "1.11.0",
-        "jshint": "2.1.4",
-        "load-grunt-tasks": "0.2.0",
-        "phantomjs": "1.9.13",
-        "q": "0.9.2",
-        "semver": "^4.1.0",
-        "jshint": "2.1.4",
-        "xmldoc": "^0.1.2",
-        "grunt-cleanempty": "1.0.3"
-    },
-    "scripts": {
-        "postinstall": "grunt install",
-        "test": "grunt test && grunt build-browser-compressed"
-    },
-    "licenses": [
-        {
-          "type": "MIT",
-          "url": "https://github.com/adobe/brackets/blob/master/LICENSE"
-        }
-    ]
+  "name": "Brackets",
+  "version": "1.4.0-0",
+  "apiVersion": "1.4.0",
+  "homepage": "http://brackets.io",
+  "issues": {
+    "url": "http://github.com/adobe/brackets/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/brackets.git",
+    "branch": "",
+    "SHA": ""
+  },
+  "dependencies": {
+    "autoprefixer-core": "5.1.8",
+    "grunt": "0.4.5",
+    "grunt-cli": "0.1.9",
+    "grunt-contrib-clean": "0.4.1",
+    "grunt-contrib-concat": "0.3.0",
+    "grunt-contrib-compress": "git://github.com/sedge/grunt-contrib-compress.git#fix-gruntjs#61",
+    "grunt-contrib-copy": "0.4.1",
+    "grunt-contrib-cssmin": "0.6.0",
+    "grunt-contrib-htmlmin": "0.1.3",
+    "grunt-contrib-jasmine": "0.4.2",
+    "grunt-contrib-jshint": "0.6.0",
+    "grunt-contrib-less": "0.8.2",
+    "grunt-postcss": "0.3.0",
+    "grunt-contrib-requirejs": "0.4.1",
+    "grunt-contrib-uglify": "0.2.0",
+    "grunt-contrib-watch": "0.4.3",
+    "grunt-git": "^0.3.4",
+    "grunt-jasmine-node": "0.1.0",
+    "grunt-lib-phantomjs": "0.3.0",
+    "grunt-npm": "git://github.com/sedge/grunt-npm.git#branchcheck",
+    "grunt-shell": "^1.1.2",
+    "grunt-targethtml": "0.2.6",
+    "grunt-template-jasmine-requirejs": "0.1.0",
+    "grunt-text-replace": "^0.4.0",
+    "grunt-update-submodules": "^0.4.1",
+    "grunt-usemin": "0.1.11",
+    "habitat": "^3.1.2",
+    "jasmine-node": "1.11.0",
+    "jshint": "2.1.4",
+    "load-grunt-tasks": "0.2.0",
+    "phantomjs": "1.9.13",
+    "q": "0.9.2",
+    "semver": "^4.1.0",
+    "xmldoc": "^0.1.2",
+    "grunt-cleanempty": "1.0.3"
+  },
+  "scripts": {
+    "postinstall": "grunt install",
+    "test": "grunt test && grunt build-browser-compressed",
+    "start": "live-server --port=8000 src"
```

Comment:
There are 2 ways to run this, from `src/` like you do here, but also from `/dist` if you've previously called `grunt build-browser`.  I wonder if we should have 2 versions of "start" somehow?  Depending on what you're doing, you'll want to be able to run both.  Not sure if it's worth adding that here, but just a thought.